### DISCUSSION TITLE
Link to the AIO web interface gets wrong when using ssh tunnel to login

### DISCUSSION THREAD

**Discussion by Aquariu** [nextcloud/all-in-one#3712] [Questions] [answered]
### Steps to reproduce
1. Shut down AIO and turn On the master container only. (using Portainer in my case)
2. Use "ssh -L" tunneling feature to access the Master container interface remotely (on the browser, the address looks like http://localhost:54321 which is redirected through ssh to the server IP
3. Perform backup, upgrade (of the master container), and restart NC AIO (with NC upgrade)
4. Login to Nextcloud once more
5. Go the Administration parameters

### Expected behavior 
The button that links to the AIO interface should lead to the correct local server IP;port

### Actual behavior 
the buttin links to http://localhost:54321/api/auth/getlogin/xxxxxxxxxxx

### Host OS 
Debian 12 VM on proxmox

#### Nextcloud AIO version 
7.4.1 (upgraded later to 7.5.1)
#### Current channel 
Latest

#### Other valuable info 
After another upgrade done locally (with direct access to the LAN where the server sits), the button that links to the AIO interface in the Admin parameters is back to normal.
I did see in the Readme that this may be considered a feature and not a bug depending on how you read it. I understand the correct behaviour to be "the link to the AIO interface should be the one exposed by the Master container". But maybe the intended behaviour is "the link to the AIO interface is the one you use on your browser when you start the containers".
Sorry if this is a feature and not a bug.
CHeers!

**Comment by szaimen** [comment, marked as answer]
> Sorry if this is a feature and not a bug.

It indeed is
